siesta-jackson/src/main/java/org/sonatype/sisu/siesta/jackson/ObjectMapperProvider.java

```diff
@@ -38,10 +37,6 @@ public ObjectMapperProvider() {
     // Configure Jackson annotations only, JAXB annotations can confuse and produce improper content
     mapper.getDeserializationConfig()
         .with(new JacksonAnnotationIntrospector());
-    // do not write null values
-    mapper.getSerializationConfig()
-        .with(new JacksonAnnotationIntrospector())
-        .withSerializationInclusion(Include.NON_NULL);
```

Comment:
AFAICT only the `withSerializationInclusion` method was removed, there's no need to drop the annotation introspector.

It also looks like the new approach to ignore null values when serializing is to use `setSerializationInclusion` which ends up calling `setDefaultPropertyInclusion`, for example:
```
    // Configure Jackson annotations only, JAXB annotations can confuse and produce improper content
    mapper.getDeserializationConfig()
        .with(new JacksonAnnotationIntrospector());
    mapper.getSerializationConfig()
        .with(new JacksonAnnotationIntrospector());

    // do not write null values
    mapper.setSerializationInclusion(Include.NON_NULL);
```